pyopal/_opal.pyi

```diff
@@ -97,7 +97,7 @@ class Database(typing.MutableSequence[str]):
         *,
         gap_open: int = 3,
         gap_extend: int = 1,
-        mode: Literal["end"] = "end",
```

Comment:
This is not a mistake -- this is to indicate that the `search` method will return an `EndResult` when the mode is `"end"`, and  a `FullResult` when the mode is `"full"`; that's how the Python standard library does it to type differently the return type of `open(..., mode="r")` and `open(..., mode="rb")`